app/services/user/user.service.go

```diff
@@ -0,0 +1,90 @@
+package user
+
+import (
+	"context"
+	model "database-ms/app/models"
+	"database-ms/config"
+	"database-ms/utils"
+
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type UserServiceInterface interface {
+	Create(context.Context, *model.User) error
+	// FindByThingId(context.Context, string) ([]*model.User, error)
+	Update(context.Context, string, *model.User) error
+	Delete(context.Context, string) error
+	GetUsers(context.Context, string) ([]*model.User, error)
+}
+
+type UserService struct {
+	db     *mgo.Session
+	config *config.Configuration
+}
+
+func NewUserService(db *mgo.Session, c *config.Configuration) UserServiceInterface {
+	return &UserService{config: c, db: db}
+}
+
+func (service *UserService) Create(ctx context.Context, user *model.User) error {
+	user.ID = bson.NewObjectId()
+	return service.addUser(ctx, user)
+}
+
+func (service *UserService) FindUpdatedUser(ctx context.Context, thingId string, lastUpdate int64) ([]*model.User, error) {
```

Comment:
I don't think it is necessary to have a service to check for updated users since we do not keep track on when they were last updated

Reply:
That's true, I'll need to be certain about what DB operations user needs.